scwx-qt/source/scwx/qt/settings/settings_interface.cpp

```diff
@@ -49,9 +50,14 @@ class SettingsInterface<T>::Impl
    std::unique_ptr<QObject> context_ {std::make_unique<QObject>()};
    QWidget*                 editWidget_ {nullptr};
    QAbstractButton*         resetButton_ {nullptr};
+   QLabel*                  unitLabel_ {nullptr};
 
    std::function<std::string(const T&)> mapFromValue_ {nullptr};
    std::function<T(const std::string&)> mapToValue_ {nullptr};
+
+   double unitScale_ {1};
+   const std::string * unitAbbreiation_ {nullptr};
```

Comment:
Spelling: `unitAbbreiation_` -> `unitAbbreviation_`

I would recommend making this just a `std::string` (or a `std::optional<std::string>`). I suspect making this a pointer here could cause issues if the function setting the value passed in a variable from the stack, unless there's some other guarantee.